meta/src/grammar.pest

```diff
@@ -6,42 +6,64 @@
 // license <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. All files in the project carrying such notice may not be copied,
 // modified, or distributed except according to those terms.
+//! Pest meta-grammar
 
+/// The top-level rule of the meta-grammar.
```

Comment:
This is a bit pedantic, but shouldn't "meta" be removed he and below? The rules are passing a normal grammar after all?